.travis.yml

```diff
@@ -6,29 +6,45 @@ os:
   - windows
 julia:
   - 1.0  # LTS
-  - 1.3  # Latest release
+  - 1    # Latest release
   - nightly
 arch:
   - x64
   - x86
 env:
-  global:
-    - PYTHON=""
-    - CONDA_JL_VERSION="3"  # Documentation stage will use the first variable set
+  # Note: Explicitly included jobs (the "Documentation stage) will use this set of variables
+  - PYTHON="" CONDA_JL_VERSION="3"
```

Comment:
```suggestion
  - PYTHON="" 
  - CONDA_JL_VERSION="3"
```

Reply:
This means something different to Travis: https://docs.travis-ci.com/user/environment-variables/#defining-multiple-variables-per-item